mkosi/__init__.py

```diff
@@ -4029,34 +4029,44 @@ def run_shell(args: Args, config: Config) -> None:
 
 
 def run_systemd_tool(tool: str, args: Args, config: Config) -> None:
-    if config.output_format not in (OutputFormat.disk, OutputFormat.directory):
+    if config.output_format not in (OutputFormat.disk, OutputFormat.directory) and not config.forward_journal:
         die(f"{config.output_format} images cannot be inspected with {tool}")
 
     if (
         args.verb in (Verb.journalctl, Verb.coredumpctl)
         and config.output_format == OutputFormat.disk
+        and not config.forward_journal
         and os.getuid() != 0
```

Comment:
Looking at this, shouldn't we check whether the user is in `systemd-journal` group as an alternative to the root check (for the `journalctl` verb)?

Reply:
@behrmann This check is about being able to mount the disk image